### PR TITLE
[ENG-828] Send Python logs to ak server logs

### PR DESCRIPTION
**Pull request by tebeka**

### CODE REVIEW

**Comment by tebeka** [comment]
Bad merge, will do in another.

**Review by daabr — COMMENTED** [review]
Doesn't it make more sense to send Python logs to the session log instead of the server log?

**Comment by tebeka** [comment]
> Doesn't it make more sense to send Python logs to the session log instead of the server log?

These are the logs from `ak_runner.py`, not the user logs.

File: runtimes/pythonrt/pythonrt.go [hunk 1]
```diff
@@ -280,22 +311,34 @@ func (py *pySvc) Run(
...
+		}
 
-	msg, err := py.comm.Recv()
-	if err != nil {
-		py.log.Error("initial message from python", zap.Error(err))
-		return nil, err
-	}
+		if msg.Type == "" {
+			py.log.Error("python can't load module", zap.String("module", mainPath))
+			return nil, fmt.Errorf("python can't load %q", mainPath)
```
Comment (efiShtain) [inline runtimes/pythonrt/pythonrt.go:324]: why do you log and turn the exact same error ?

**Review by efiShtain — APPROVED** [review]

File: runtimes/pythonrt/pythonrt.go (hunk 1 above)
Comment (tebeka) [reply runtimes/pythonrt/pythonrt.go:324]: Not sure how `ak` logs it, helps me with debugging.

**Pull request merged**